SPECS/verity-read-only-root/create_linear_debug_mount.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
nit: is it of any significance that some of the scripts invoke bash while others invoke sh?

Reply:
Yes, the scripts that run in the initramfs (not sure if by necessity or convention) all run /bin/sh, while the module-setup.sh which runs in the context of the full OS uses bin/bash.

---
In reply to: [552693738](https://github.com/microsoft/CBL-Mariner/pull/483#discussion_r552693738) [](ancestors = 552693738)